src/parquet/arrow/writer.cc

```diff
@@ -717,6 +717,10 @@ Status FileWriter::WriteTable(const Table& table, int64_t chunk_size) {
     }
   }
 
+  if (chunk_size <= 0) {
+    return Status::Invalid("chunk size per row_group must be greater than 0");
```

Comment:
Invalid Status or DCHECK? I prefer Invalid Status

Also, is a test needed? I prefer not. 
@wesm